### PR TITLE
Doc

### PR DESCRIPTION
**Pull request by empEvil**
Added a Sphinx documentation folder, and setup it up such that it will look through the files

### CODE REVIEW

**Comment by beckermr** [comment]
Please remove the _build dir from the commit.

**Comment by empEvil** [comment]
should be removed now

File: documentation/api/firecrown.analysis.rst [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+
+    firecrown.analysis.tests
+
+Submodules
+----------
+
+.. toctree::
+
+   firecrown.analysis.emcee_interface
+   firecrown.analysis.test_loglike
```
Comment (beckermr) [inline documentation/api/firecrown.analysis.rst]: We should convince sphinx to ignore the test directories.

**Comment by empEvil** [comment]
https://tjpcosmo.readthedocs.io/en/doc/

**Comment by empEvil** [comment]
@beckermr I think I fixed, the tests things, but we might want to set up a new Readthedocs webpage.

**Comment by beckermr** [comment]
Yup it should be called firecrown!

**Comment by beckermr** [comment]
Feel free to set it up and then add me as a maintainer

**Comment by empEvil** [comment]
Made a new Read the docs, its called DESCfirecrown since I couldn't seem to use the name firecrown. Should the doc part be merged into master, so it's master it generates the autodoc from?

**Comment by beckermr** [comment]
Yep. Let's merge and then build latest from master.

**Comment by empEvil** [comment]
I did add you as maintainer on readthedocs

Get Outlook for iOS<https://aka.ms/o0ukef>

________________________________

**Pull request merged**